### PR TITLE
Replaces starboard with trivy-operator library

### PR DESCRIPTION
**Pull request by rewanthtammana**
### Description

* Replaces starboard with trivy-operator library
* Removes CIS benchmarks & reporting capabilities

### Checklist

- [x] Test to make sure the vulnerability reports are generated.
- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.
- [ ] (Giant Swarm) If creating a release, bump the `version` and `appVersion` in Chart.yaml.

### CODE REVIEW

File: controllers/vulnerabilityreport/vulnerabilityreport_controller.go [hunk 1]
```diff
@@ -194,8 +194,9 @@ func getCountPerSeverity(report *aqua.VulnerabilityReport) map[string]float64 {
 		string(aqua.SeverityHigh):     float64(report.Report.Summary.HighCount),
 		string(aqua.SeverityMedium):   float64(report.Report.Summary.MediumCount),
 		string(aqua.SeverityLow):      float64(report.Report.Summary.LowCount),
-		string(aqua.SeverityNone):     float64(report.Report.Summary.NoneCount),
-		string(aqua.SeverityUnknown):  float64(report.Report.Summary.UnknownCount),
+		// aqua.SeverityNone was in starboard but removed in trivy-operator
+		// string(aqua.SeverityNone):    float64(report.Report.Summary.NoneCount),
```
Comment (stone-z) [inline controllers/vulnerabilityreport/vulnerabilityreport_controller.go:198]: Maybe we can just remove it then. We'll just have to test this with existing VulnerabilityReports and see if they can still be reconciled.

**Review by stone-z — COMMENTED** [review]
Not sure if you meant to request a review yet but this is looking really good so far

**Comment by rewanthtammana** [comment]
I didn't get to test it yet, so I put in draft state. I completed the testing now & everything is working as intended.

On CLI: 

image


On grafana:

image

**Comment by rewanthtammana** [comment]
Closes https://github.com/giantswarm/giantswarm/issues/22754

**Review by stone-z — APPROVED** [review]
Nice :rocket:

**Pull request merged**